api.ts

```diff
@@ -2151,9 +2153,10 @@ export function makeSyncTable<
     });
   }
 
-  const formulaSchema = getSchema
+  const normalizedSchema = normalizeSchema(schema);
```

Comment:
this is to make sure we don't call normalize twice on the same schema

Reply:
I added some tests, including a bug fix I found by using this SDK version in the packs repo.